stacks/SiriGeneratorStack.ts

```diff
@@ -1,17 +1,22 @@
 import { TreatMissingData } from "aws-cdk-lib/aws-cloudwatch";
 import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import { SubnetType } from "aws-cdk-lib/aws-ec2";
 import { PolicyStatement } from "aws-cdk-lib/aws-iam";
 import { EventType, Bucket as S3Bucket } from "aws-cdk-lib/aws-s3";
 import { LambdaDestination } from "aws-cdk-lib/aws-s3-notifications";
 import { Cron, Function, StackContext, use } from "sst/constructs";
 import { DynamoDBStack } from "./DynamoDBStack";
 import { MonitoringStack } from "./MonitoringStack";
+import { RdsStack } from "./RdsStack";
+import { VpcStack } from "./VpcStack";
 import { createBucket } from "./utils";
 
 export const SiriGeneratorStack = ({ stack }: StackContext) => {
     const { disruptionsTable, organisationsTableV2: organisationsTable } = use(DynamoDBStack);
```

Comment:
```suggestion
    const { organisationsTableV2: organisationsTable } = use(DynamoDBStack);
```

References to disruptionsTable for permissions and env vars can be removed from the the function in this file.